### ISSUE TITLE
Consider moving project specific tests elsewhere (RFC)

### ISSUE THREAD

**Issue by ZenGround0** [ipfs/kubernetes-ipfs#36]
Not at all urgent, but if kubernetes-ipfs starts to see regular use by multiple projects it will make less sense for project specific tests to be kept in this repo.  Welcome thoughts from @FrankPetrilli and @hsanjuan about restructuring in the future.

**Comment by hsanjuan** [comment]
Agreed, but not urgent. I'm not super bothered by it at this point.
